### DISCUSSION TITLE
v2.5.3

### DISCUSSION THREAD

**Discussion by rdbende** [rdbende/Sun-Valley-ttk-theme#102] [Announcements]
If you're using sv_ttk >=2.5.0,<2.5.3, PLEASE UPDATE `sv_ttk` NOW!

Fixed #100

**Comment by littlewhitecloud** [comment]
XD
![image](https://github.com/rdbende/Sun-Valley-ttk-theme/assets/71159641/f3c9caff-aaf9-4dbb-9b1b-340f5b131288)

**Comment by lg-nineoneninethreenine** [comment]
@rdbende you have to update the release. It says "Fugbix" instead of "Bugfix".

**Reply by rdbende** [reply]
That was intentional :P

**Reply by lg-nineoneninethreenine** [reply]
But why? What's it mean?

**Reply by rdbende** [reply]
It means that I'm funny :D

**Reply by lg-nineoneninethreenine** [reply]
😂
